lib/ThreeScaleClient.php

```diff
@@ -257,17 +270,24 @@ public function authorize_with_user_key($userKey, $serviceId = null, $usage = nu
    * </code>
    */
 
-   public function authrep($appId, $appKey = null, $usage = null, $userId = null, $object = null, $no_body = null, $serviceId = null) {  
+   public function authrep($appId, $appKey = null, $options, $usage = null, $userId = null, $object = null, $no_body = null) {  
```

Comment:
So this is a breaking change. I guess we could mention it in the change log that the signature changed?